src/PrestaShopBundle/Controller/Admin/TranslationsController.php

```diff
@@ -49,6 +49,8 @@ class TranslationsController extends FrameworkBundleAdminController
     const controller_name = self::CONTROLLER_NAME;
 
     /**
+     * @AdminSecurity("is_granted(['read'], request.get('_legacy_controller'))")
```

Comment:
I dont know whether the fact it was not secured is intentional however the returned content seems quite harmless. I dont know :/

Reply:
are there any downsides of securing it?

Reply:
Yes, if this ajax route is used to retrieve informations from other page.

For example if page Module calls this ajax route to display some informations, then a user who is granted only READ access to module page needs to have also translations page READ access to display the Module page correctly. Before, as it was unsecured ... it worked 😅

Reply:
true, this is tricky :thinking:  but on the other hand, if all other pages are secured the same way, why should we skip security on this one?

Reply:
It's rather than this is a page that was migrated in the early days of PS 1.7 when the migration plan was not clear nor clean, so it might have been unsecured on purpose and I'm afraid of breaking something that worked (and was unsecured on purpose by design).

Reply:
so should I remove the security from this action?

Reply:
I think it's better for now indeed

Reply:
removed it